### DISCUSSION TITLE
Yourls roadmap and future update

### DISCUSSION THREAD

**Discussion by faselunare** [YOURLS/YOURLS#2927] [General]
Hi, I would like to know how much Yourls is maintained and if there is a roadmap. 
I ask because I am working on a custom theme and one or more plugins.

**Comment by ozh** [comment]
How much YOURLS is maintained : very maintained. Such project. Much love. More seriously you can get a good grasp at the dev pace just by looking at the commits, releases, [pulse](https://github.com/YOURLS/YOURLS/pulse), etc...

Roadmap : not so much to be honest :)

Your developing plugins : go for it. The plugin API is what really makes YOURLS awesome, so no roadmap milestone should break it.

**Reply by faselunare** [reply]
ok, i will continue with my 'yourls' development!
